flow/layers/physical_shape_layer_unittests.cc

```diff
@@ -202,5 +202,74 @@ TEST_F(PhysicalShapeLayerTest, ElevationComplex) {
                0, MockCanvas::DrawPathData{layer_path, layer_paint}}}));
 }
 
+#define MAKE_PHYSICAL_SHAPE_LAYER(clip)                              \
```

Comment:
For that one, that is reasonable, but for the others, they refer to info that is not available in a static method and so I made them macros to avoid having to pass in a huge argument list:

Here is the context you can't fetch on your own
And here is the SkMatrix I already created for you (arguably could be created every time in the called method)
And here is the path I made for the clip (again, could be created every time)
And here is the (mock) child I made which may or may not do readback.
Etc.

Some of them could just be created locally on every call, though...

Reply:
Done.